site/gdocs/BlockErrorBoundary.tsx

```diff
@@ -1,48 +1,57 @@
 import React from "react"
 import { ErrorBoundary } from "react-error-boundary"
+import { getLayout } from "./ArticleBlock.js"
 import { useDebug } from "./DebugContext.js"
 
 export const BlockErrorBoundary = ({
     children,
 }: {
     children: React.ReactNode
 }) => {
-    const debug = useDebug()
-    return debug ? (
+    return (
         <ErrorBoundary FallbackComponent={BlockErrorFallback}>
             {children}
         </ErrorBoundary>
-    ) : (
-        <>{children}</>
     )
 }
 
 export const BlockErrorFallback = ({
     error,
     resetErrorBoundary,
-    className = "",
+    className = getLayout(),
```

Comment:
What's the motivation for this change? It's better than `""` I suppose, but calling `getLayout` like this will _always_ return the `layouts[default][default]` layout, which means that if it's in a column that's less than 12 wide, it'll break the layout.

Reply:
Thanks! Yes the error was a bit squished in e.g. sticky layouts since it would start at col 5 (and stop at the edge of the grid). This might not be entirely obvious but this only affects client-side errors - so testing this requires throwing on the client only (e.g. `if (typeof window !== "undefined") throw new Error("This is a test error")` in Chart.tsx)